git-publish

```diff
@@ -952,6 +952,11 @@ branch.%s.pushRemote is set appropriately?  (Override with --no-check-url)''' %
             print(e)
             return 1
         finally:
+            if options.dry_run:
```

Comment:
Why is this in the `finally:` body that gets executed even when an error occurs? I think this should go after the `git_send_email()` calls inside the `try:` body since the output should only be displayed in the success case.

Reply:
Ahh I thought finally was for all the exit paths (you clean up tmpdir there as well)